### PR TITLE
[CL-111] remove h3 from bit-callout

### PR DESCRIPTION
**Pull request by cd-bitwarden**
## Type of change

```
- [ x ] Bug fix
- [ ] New feature development
- [ ] Tech debt (refactoring, code cleanup, dependency upgrades, etc)
- [ ] Build/deploy pipeline (DevOps)
- [ ] Other
```

## Objective

Stop A11y from complaining about H3 tag, instead update component to not use H3 since it doesn't make sense
## Code changes

libs/angular/src/components/callout.component.html - changing div to aside, and h3 to header to maintain look but stop A11y from complaining

- **file.ext:** Description of what was changed and why

## Screenshots

image

- Please add **unit tests** where it makes sense to do so (encouraged but not required)
- If this change requires a **documentation update** - notify the documentation team
- If this change has particular **deployment requirements** - notify the DevOps team
- Ensure that all UI additions follow [WCAG AA requirements](https://contributing.bitwarden.com/contributing/accessibility/)

### CODE REVIEW

File: libs/angular/src/components/callout.component.html
Comment (willmartian) [inline libs/angular/src/components/callout.component.html:1]: The actual changes look good, but this is the wrong component. The callout used in SM is the `bit-callout` component, which [is part of the CL](https://github.com/bitwarden/clients/blob/master/libs/components/src/callout/callout.component.html). 

(If you want to change `app-callout` as well, make sure to double check this change doesn't introduce any visual diffs. At a quick glance, you would need to update any css like `h3.callout-heading`)

**Review by willmartian — CHANGES_REQUESTED** [review]

**Comment by cd-bitwarden** [comment]
image

**Review by willmartian — APPROVED** [review]
Looks good! Thanks 💯

**Review by willmartian — CHANGES_REQUESTED** [review]
Hey @cd-bitwarden,

Was chatting with @danielleflinn about this ticket, and she pointed out that `aside` elements also need a unique id on their label/heading. (good catch, thanks!)

Here is [an example](https://github.com/bitwarden/clients/blob/master/libs/components/src/dialog/directives/dialog-title-container.directive.ts#L11) of how we do so in `bit-dialog`. We should put an `id` on the `header`, and then reference it with `aria-labelledby` on the `aside`. 

Please find more info here: https://www.w3.org/WAI/ARIA/apg/patterns/landmarks/examples/complementary.html

**Review by willmartian — APPROVED** [review]
LGTM

**Comment by cd-bitwarden** [comment]
Closing PR because Will created a PR that has these changes

**Comment by willmartian** [comment]
I think this one was closed by mistake. Reopening! 🌈

**Pull request merged**